spring-cloud-commons/src/main/java/org/springframework/cloud/commons/httpclient/HttpClientConfiguration.java

```diff
@@ -33,8 +33,34 @@
 public class HttpClientConfiguration {
 
 	@Configuration(proxyBeanMethods = false)
-	@ConditionalOnProperty(name = "spring.cloud.httpclientfactories.apache.enabled", matchIfMissing = true)
+	@ConditionalOnProperty(name = "spring.cloud.httpclientfactories.apache5.enabled", matchIfMissing = true)
```

Comment:
I probably leave `apache`

Reply:
Oh, you've got both?

Reply:
Right implementations for both, depending on whats on the classpath

Reply:
They would have to put apache4 on the classpath AND enable to property enabled